Assets/Mona/TemplateScripts/PlayerProperties.cs

```diff
@@ -0,0 +1,39 @@
+using UnityEngine;
+
+namespace Mona
+{
+    [System.Serializable]
+    public class PlayerProperties
+    {
+        // Fall speed of players in this volume
+        [Tooltip("Fall speed of players in this volume")]
+        public float Gravity = -15f;
```

Comment:
Can you test how editing the X and Z vectors for gravity? If they work well, we should include them altogether in a `Vector3` class instead of a single float. It might be cool if someone wants to make an [O'Neill Cylinder](https://astronomy.com/~/media/8F29E55B369C450D92B9FFF2A4D35B6D.jpg) or something like that.